docs/commands/push.md

```diff
@@ -324,3 +329,136 @@ The Redocly Workflows interface can help you get started with the `push` command
+| --mount-path, -mp     |  string  | true     | -                                                              | The path where the files are mounted in the project.                                                                                                       |
+| --branch, -b          |  string  | true     | -                                                              | The branch files are pushed from.                                                                                                                          |
+| --author, -a          |  string  | true     | -                                                              | The author of the push in the format: `'Author Name <author-email@example.com>'`.                                                                          |
+| --message, -m         |  string  | true     | -                                                              | The commit message for the push.                                                                                                                           |
+| --commit-sha, -sha    |  string  | false    | -                                                              | Commit SHA.                                                                                                                                                |
+| --commit-url, -url    |  string  | false    | -                                                              | Commit URL.                                                                                                                                                |
+| --repository          |  string  | false    | -                                                              | Repository ID. Example: `redocly-cli`.                                                                                                                     |
+| --namespace           |  string  | false    | -                                                              | Repository owner/organization/workspace. Example: `Redocly`.                                                                                               |
+| --created-at          |  string  | false    | -                                                              | Commit creation date. Format: `yyyy-mm-ddThh:mm:ss+offset value`. Example: `2024-02-20T14:26:26+02:00`                                                     |
+| --domain              |  string  | false    | [https://app.cloud.redocly.com](https://app.cloud.redocly.com) | The domain to which the files are pushed.                                                                                                                  |
+| --default-branch      |  string  | false    | main                                                           | The default branch of the repository the push originates from.                                                                                                          |
```

Comment:
I wasn't real sure how this "default branch" differs from the "branch" option.

Reply:
It's the branch on the origin repository, rather than the one we're publishing to

Reply:
I can't figure out how to say that in different words that isn't more confusing ...